chore(deps): bump dependency versions

adrianblazquez opened the pull request:
Update [ablazleon/myserver](https://github.com/ablazleon/myserver.git) 

Command run was `jx create quickstart --git-public=true`
<hr />

Update [ablazleon/my-server](https://github.com/ablazleon/my-server.git) 

Command run was `jx create quickstart --git-public=true`
<hr />

Update [ablazleon/ml-api-docker-k8](https://github.com/ablazleon/ml-api-docker-k8.git) 

Command run was `jx import --url https://github.com/ablazleon/ml-api-docker-k8.git`
<hr />

Update [ablazleon/udagram-frontend](https://github.com/ablazleon/udagram-frontend.git) 

Command run was `jx import --url https://github.com/ablazleon/udagram-frontend.git`
<hr />

Update [ablazleon/myexample](https://github.com/ablazleon/myexample.git) 

Command run was `jx create quickstart`

adrianblazquez committed:
chore(deps): bump https://github.com/ablazleon/myexample.git

Command run was `jx create quickstart`

ablazleon commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **ablazleon**
You can assign the PR to them by writing `/assign @ablazleon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ablazleon%2Fenvironment-myinfra-dev).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ablazleon/environment-myinfra-dev/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["ablazleon"]} -->

ablazleon commented:
@ablazleon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
promotion-build | 76844f505bd6d075d5c5cc35d405f8feddba079a | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adrianblazquez committed:
chore(deps): bump https://github.com/ablazleon/udagram-frontend.git

Command run was `jx import --url https://github.com/ablazleon/udagram-frontend.git`

ablazleon commented:
@ablazleon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
promotion-build | ebd5d9feb4a84f7e611e4301d694336ab9151774 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adrianblazquez committed:
chore(deps): bump https://github.com/ablazleon/ml-api-docker-k8.git

Command run was `jx import --url https://github.com/ablazleon/ml-api-docker-k8.git`

ablazleon commented:
@ablazleon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
promotion-build | 7f5e187296547bf1d6d9adbaaf63207a02e7cb07 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adrianblazquez committed:
chore(deps): bump https://github.com/ablazleon/my-server.git

Command run was `jx create quickstart --git-public=true`

ablazleon commented:
@ablazleon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
promotion-build | fe6e54d8e99bb072dae8e4bf4ee9726f48f21846 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

adrianblazquez committed:
chore(deps): bump https://github.com/ablazleon/myserver.git

Command run was `jx create quickstart --git-public=true`

ablazleon commented:
@ablazleon: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
promotion-build | 2027a1e5b1e8168307330b213f5ce71886104749 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->
